rollup.config.js

```diff
@@ -29,6 +29,37 @@ const umd_config = {
   ],
 };
 
+const cli_config = {
+  onwarn(message) {
+    if (message.code === "EVAL") { return; }
```

Comment:
Why this needed? If I understand correctly, this code suppresses warnings about `eval(...)` usage, but `eval` not used in the CLI code. I tried to build with this line commented and did not notice any warnings.

It is better to add a comment describing what exactly happens here

Reply:
Removed for the CLI.